gatsby-node.js

```diff
@@ -4,7 +4,7 @@ const path = require('path');
 async function createUserStoryPages({graphql, createPage, createRedirect}) {
     const userStory = path.resolve('src/pages/_user_story.jsx');
     const result = await graphql(`{
-        stories: allUserStory {
+        stories: allUserStory (sort: { fields: [date], order: DESC }) {
```

Comment:
```suggestion
        stories: allUserStory (sort: {date: DESC}) {
```

The syntax you proposed is deprecated. Did you run lint to check?